src/urllib3/util/ssl_.py

```diff
@@ -397,6 +404,19 @@ def ssl_wrap_socket(
     return context.wrap_socket(sock)
 
 
+def has_alpn(ctx_cls=None):
```

Comment:
I'm not sure if this function is needed by users since ALPN both isn't configurable and is also "opportunistic" ie not the end of the world if it actually gets set or not. If this function is only used within the test suite let's remove it from `urllib3.util` and add it to test utilities.

Reply:
Done, moved to ```test/__init__.py```.